content/source/docs/enterprise/api/team-access.html.md

```diff
@@ -73,15 +71,20 @@ $ curl \
 
 ## Add Team access to a Workspace
 
-| Method | Path           |
-| :----- | :------------- |
-| POST | /team-workspaces |
+`POST /team-workspaces`
 
-### Parameters
+### Request Body
 
-- `access` (`string: <required>`) - `read`, `write`, or `admin`
-- `team` (`string: <required>`) - The ID of the team to add to the workspace.
-- `workspace` (`string: <required>`) - The workspace ID to which the team is to be added.
+This POST endpoint requires a JSON object with the following properties as a request payload. (Properties without a default value are required.)
+
+Key path                                 | Type   | Default | Description
```

Comment:
I'm not sure what editor you use, but if you have `prettier` or another tool like it, you can have it format your Markdown tables automatically.

Reply:
I've got a plugin script that handles that for me. I don't think this table is mis-formatted? Or is it?

Reply:
Sorry, meant as a general tip. Not specific to this case.